src/pages/pages/integrations.md

```diff
@@ -0,0 +1,44 @@
+---
+templateKey: default-page
+seo:
+  description: "Integrations for Ironic Bare Metal"
+  title: Ironic Bare Metal Integrations
+  twitterUsername: "@openstack"
+  url: "https://ironicbaremetal.org/"
+title: Integrations
+subTitle: "Ironic project Integrations and Tools"
+---
+
+The Ironic project has a long history of being driven by Cloud and Infrastruture operators seeking to solve their problems. This has resulted in a diverse set of capabilities, integrations, and ways to leverage ironic.
+
+<br>
+
+## Community Projects with Integration
+
+This is a list of community projects which feature or leverage Ironic in order to deploy Bare Metal to meet the needs of infrastucture operators and ultimately end users. Some of these projects have commercial versions or offerings from vendors, however this list is restricted to "Community" only.
+
+- **Bifrost:** <https://docs.openstack.org/bifrost/latest/install/index.html> - An Ansible based toolkit to setup Ironic, and orchustrate Bare Metal with Playbooks!
+- **TripleO:** <https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/> - A toolkit and suite of projects which use Ironic to deploy the Bare Metal and offer the ability to deploy Bare Metal as a Service level configurations. Able to be used to deploy an entire OpenStack cluster.
+- **Kayobe:** <https://docs.openstack.org/kayobe/latest/> - A toolkit to deploy an OpenStack cluster using physical baremetal which leverages Ironic to perform Bare Metal deployment.
+- **Metal³:** <https://metal3.io> - A toolkit to help facilitate baremetal machine deployment from with-in a Kubernetes deployment, leveraging Ironic in an an ephemeral design.
```

Comment:
Inconsistent spelling: "baremetal" and "Bare Metal", please pick one

Reply:
Went to two words since the module name mention will help trigger searches on "baremetal"